OSGi/bundle-with-package-dependency/pom.xml

```diff
@@ -0,0 +1,50 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>fish.payara</groupId>
+    <artifactId>bundle-example-eclipselink</artifactId>
+    <version>1.0</version>
+    <packaging>bundle</packaging>
+
+    <name>Payara-Example-Bundle</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.core</artifactId>
+            <version>4.3.0</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.persistence</groupId>
+            <artifactId>eclipselink</artifactId>
+            <version>2.6.4</version>
+            <scope>provided</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>2.3.7</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-Activator>fish.payara.bundle.Activator</Bundle-Activator>
+                        <Import-Package>org.eclipse.persistence.expressions,
+                            org.osgi.framework;version="[1.5,2)"</Import-Package>
+                        <Bundle-SymbolicName>Payara-Example</Bundle-SymbolicName>
```

Comment:
It is recommended to treat symbolic name like a package (or automatic module name in recent days), therefore `fish.payara.bundle` is more fitting name

Reply:
I changed both package name and symbolic name to `fish.payara.examplebundle`